_site/2020/course-shell/index.html

```diff
@@ -0,0 +1,473 @@
+</code></pre></div></div>
+
+<div class="language-console highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="gp">missing:~$</span><span class="w"> </span><span class="nb">ls</span> <span class="nt">-l</span> /home
+<span class="go">drwxr-xr-x 1 missing  users  4096 Jun 15  2019 missing
+</span></code></pre></div></div>
+
+<p>This gives us a bunch more information about each file or directory
+present. First, the <code class="language-plaintext highlighter-rouge">d</code> at the beginning of the line tells us that
+<code class="language-plaintext highlighter-rouge">missing</code> is a directory. Then follow three groups of three characters
+(<code class="language-plaintext highlighter-rouge">rwx</code>). These indicate what permissions the owner of the file
+(<code class="language-plaintext highlighter-rouge">missing</code>), the owning group (<code class="language-plaintext highlighter-rouge">users</code>), and everyone else respectively
+have on the relevant item. A <code class="language-plaintext highlighter-rouge">-</code> indicates that the given principal does
+not have the given permission. Above, only the owner is allowed to
+modify (<code class="language-plaintext highlighter-rouge">w</code>) the <code class="language-plaintext highlighter-rouge">missing</code> directory (i.e., add/remove files in it). To
+enter a directory, a user must have “search” (represented by “execute”:
+<code class="language-plaintext highlighter-rouge">x</code>) permissions on that directory (and its parents). To list its
+contents, a user must have read (<code class="language-plaintext highlighter-rouge">r</code>) permissions on that directory. For
+files, the permissions are as you would expect. Notice that nearly all
+the files in <code class="language-plaintext highlighter-rouge">/bin</code> have the <code class="language-plaintext highlighter-rouge">x</code> permission set for the last group,
+“everyone else”, so that anyone can execute those programs.</p>
+
+<p>Some other handy programs to know about at this point are <code class="language-plaintext highlighter-rouge">mv</code> (to
+rename/move a file), <code class="language-plaintext highlighter-rouge">cp</code> (to copy a file), and <code class="language-plaintext highlighter-rouge">mkdir</code> (to make a new
+directory).</p>
+
+<p>If you ever want <em>more</em> information about a program’s arguments, inputs,
+outputs, or how it works in general, give the <code class="language-plaintext highlighter-rouge">man</code> program a try. It
+takes as an argument the name of a program, and shows you its <em>manual
+page</em>. Press <code class="language-plaintext highlighter-rouge">q</code> to exit.</p>
+
+<div class="language-console highlighter-rouge"><div class="highlight"><pre
+class="highlight"><code><span class="gp">missing:~$</span><span class="w"> </span><span class="nb">man</span> ls
```

Comment:
This was the particular line that needed the styling change. Applied class "nb" to "man" and removed styling from "ls"